incubator/harbor/README.md

```diff
@@ -0,0 +1,124 @@
+# VMWare Harbor
+
+[VMWare Harbor](http://vmware.github.io/harbor/) is an enterprise-class registry server that stores and distributes Docker images. Harbor extends the open source Docker Distribution by adding the functionalities usually required by an enterprise, such as security, identity and management. As an enterprise private registry, Harbor offers better performance and security. Having a registry closer to the build and run environment improves the image transfer efficiency. Harbor supports the setup of multiple registries and has images replicated between them. In addition, Harbor offers advanced security features, such as user management, access control and activity auditing.
+
+## Introduction
+
+This chart installs and configures VMWare Harbor.
+
+## Prerequisites
+
+- Kubernetes 1.7+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ git clone https://github.com/paulczar/helm-harbor.git harbor
+$ cd harbor
+$ helm install --name my-release .
+```
```

Comment:
The install example should be updated for the incubator repo rather than the git repo it has today. There are examples in the other incubator charts.